src/components/ImageUploader.js

```diff
@@ -0,0 +1,54 @@
+const aws = require('aws-sdk')
```

Comment:
`ImageUploader` is not a component, it should not be in the components directory if you meant it to be a js file and not a vue file.

Reply:
There is already a `UploadImage.vue` component. It's better to build on this. The change you need to make will be in the `showFileFinder` method. Here, after an image is selected, dispatch an event to do the actual upload to S3. 

```
### example:

...
methods: {
		showFileFinder() {
			$('#myfile').trigger('click');

			const input = document.querySelector('input');
			input.addEventListener('change', (e) => {
				const [file] = e.target.files;
				this.fileName = file.name
        this.$store.dispatch('upload_to_s3') //this will be handled in a state action that you must create. It can accept additional parameters if you need to. Find examples in other components.
			})
		}
	}
...
```

Reply:
Please remove ImageUploader.js from components directory, does not look like you'd be needing it. Let's build on `UploadImage.vue` as described above.